GOVERNANCE.md

```diff
@@ -141,6 +155,8 @@ The process for changing the policy is:
 * After an appropriate period of discussion (a few days), update the proposal based on feedback if required and resend it to all current committers with a request for a formal vote.
 * After all votes have been collected or a reasonable time has elapsed for them to be provided (for example, a couple of business days), the votes are evaluated. For the request to modify the policy to pass, it must receive yes votes from two thirds of the existing committers.
 
+This does not apply to editorial changes, which can be implemented via lazy consensus. This includes things like stylistic tweaks, fixing spelling or grammar, or updating links. It also includes adding implicit practices and procedures that are not yet documented. 
```

Comment:
```suggestion
This does not apply to editorial changes, which can be implemented via lazy consensus. This includes things like stylistic tweaks, fixing spelling or grammar, or updating links. 
```

Reply:
because they are implicit not everyone may know or agree about them so better to have a discussion when making them explicit